### PR TITLE
Test: add set orientation

### PR DESCRIPTION
**Pull request by TonyBrobston**
Here's the beginning of `setOrientation` if you're open to implementing it. I could look into the implementation, just figured I'd start with a test.

Relates to https://github.com/ginpei/exif-orientation/issues/15

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -76,6 +76,13 @@ export async function getOrientation (
   return info;
 }
 
+export async function setOrientation (
+  input: File | Buffer | ArrayBuffer,
+  orientation: number,
```
Comment (ginpei) [inline src/index.ts]: It would be better to have this `orientation` type `OrientationCode | IOrientationInfo`

Comment (TonyBrobston) [reply src/index.ts]: I swapped to `OrientationCode` for now. I think I'll need a new test for to drive myself into adding `IOrientationInfo`.

Comment (ginpei) [reply src/index.ts]: Sorry I now think `IOrientationInfo` is a little bit too much. Only `OrientationCode`  would be great.

Comment (TonyBrobston) [reply src/index.ts]: I was thinking the same thing

File: src/index.ts [hunk 2]
```diff
@@ -116,6 +114,42 @@ export async function readOrientationCode (
...
+  const littleEndian = isLittleEndian(view, tiffHeaderOffset);
+  const ifdPosition = findIfdPosition(view, tiffHeaderOffset, littleEndian);
+  const ifdFieldOffset = ifdPosition + statics.ifdFieldCountLength;
+  const orientationOffset = findOrientationOffset(
+    view,
+    ifdFieldOffset,
+    littleEndian,
+  );
+  return {littleEndian, orientationOffset};
+}
```
Comment (TonyBrobston) [inline src/index.ts:144]: I don't love this. Trying to reduce duplication.

**Comment by TonyBrobston** [comment]
@ginpei I've finished my first round of coding, let me know what changes you would like.

**Comment by TonyBrobston** [comment]
@ginpei Are you able to give me some feedback on these changes?

File: src/index.ts [hunk 3]
```diff
@@ -76,6 +76,13 @@ export async function getOrientation (
   return info;
 }
 
+export async function updateOrientation (
```
Comment (ginpei) [inline src/index.ts]: Can you delete this `updateOrientation()` function? Let's have only `updateOrientationCode()`.

The original idea of the difference between `getOrientation()` and `readOrientationCode()` is which type they return. The "code" one returns `OrientationCode` and the other returns `IOrientationInfo`. So this `updateOrientation()` should accept "info" if we implement.

I'm sorry I should've told you this before but I needed time to remember.

**Review by ginpei — CHANGES_REQUESTED** [review]

**Comment by ginpei** [comment]
I added one last request. I will merge and release package soon right after updated.

File: src/index.ts (hunk 3 above)
Comment (TonyBrobston) [reply src/index.ts]: This makes sense. No need to be sorry, I was asking for feedback :). I deleted the function as you requested.

**Comment by TonyBrobston** [comment]
I just realized I haven't updated the readme at all, is that something you would mind handling? If not, I can give it a try.

**Comment by ginpei** [comment]
Good point. Please go ahead.

**Review by ginpei — APPROVED** [review]

**Comment by TonyBrobston** [comment]
@ginpei I updated the documentation. I tried to follow your pattern.

File: README.md [hunk 4]
```diff
@@ -22,6 +22,12 @@ console.log(
 );
 ```
 
+```ts
+import * as exif from '@ginpei/exif-orientation';
+
+await exif.updateOrientationCode(fileOrBuffer, 5);
```
Comment (ginpei) [inline README.md]: Use `exif.OrientationCode.deg90Flipped` instead please

**Comment by ginpei** [comment]
@TonyBrobston Requested one thing. Looks good to me other than that.

Do you want to do anything else before merging? I will prepare release note with your name and account.

**Comment by TonyBrobston** [comment]
Done.

Everything seems good to me 👍 .

**Review by ginpei — APPROVED** [review]
Great!

**Pull request merged**